### PR TITLE
12.0 mig project_stage_closed

### PR DESCRIPTION
**Pull request by ernestotejeda**
Cc @Tecnativa

### CODE REVIEW

**Review by chienandalu — APPROVED** [review]
:+1: Tested on runbot. Thanks!

**Review by LoisRForgeFlow — APPROVED** [review]
Tested on runbot :+1:

File: project_stage_closed/tests/test_project_stage_closed.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+
+class TestProjectStageClosed(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super(TestProjectStageClosed, cls).setUpClass()
+        cls.stage = cls.env['project.stage.type'].create({
+            'name': 'Test stage',
+            'closed': True,
+        })
```
Comment (nikul-serpentcs) [inline project_stage_closed/tests/test_project_stage_closed.py]: @ernestotejeda here Test Some condition, True Or False

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Functional tests.

BTW why is it not called project_task_stage_closed? I know this was like that in previous versions, but it is a bit confusing, it is suggesting we have stages in the project.

**Comment by pedrobaeza** [comment]
@aheficent. Stages are at project level, not task. Previously, when there were issues, the same stages were used. That's why it doesn't have the `_task` surname. Now that they have gone, indeed they are the only place right now where they are used, but any way, we can leave it as is as the stages can be reused in third party modules.

File: project_stage_closed/tests/test_project_stage_closed.py (hunk 1 above)
Comment (pedrobaeza) [reply project_stage_closed/tests/test_project_stage_closed.py]: This is the setup, so no assert should be done here.

Comment (pedrobaeza) [reply project_stage_closed/tests/test_project_stage_closed.py]: Ah, OK, I got it. This is a test with no test! @ernestotejeda don't add a test if not needed, like this case.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Remove the useless test.

**Comment by AaronHForgeFlow** [comment]
@pedrobaeza Yes, stages may be used in other places but the object is called 'project.task.type'. Anyway it is not blocking IMO.

**Comment by pedrobaeza** [comment]
For that blame Odoo that didn't rename that model in the good moment :wink:

File: project_stage_closed/README.rst [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. Have Manager rights for Project group.
+#. Go to *Project > Configuration > Stages*.
```
Comment (AaronHForgeFlow) [inline project_stage_closed/README.rst:41]: Is this option still available? I seems to me this menu was removed.

Comment (chienandalu) [reply project_stage_closed/README.rst:41]: Enable debug mode :wink:

Comment (AaronHForgeFlow) [reply project_stage_closed/README.rst:41]: I wonder Odoo put all those settings under debug. Changing those settings has nothing to do with debugging...

Comment (pedrobaeza) [reply project_stage_closed/README.rst:41]: That's why we always install `base_technical_features` and enable it for all users.

Comment (chienandalu) [reply project_stage_closed/README.rst:41]: That's why OCA's `base_technical_features` exists :slightly_smiling_face:

Comment (AaronHForgeFlow) [reply project_stage_closed/README.rst:41]: hehe I would put `'auto install' = True` to that module

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**